brilirs/src/interp.rs

```diff
@@ -87,17 +91,61 @@ impl Environment {
   }
 }
 
+//A second heap for atomics, since AtomicI64 cannot be added to the Value enum without a lot of work
```

Comment:
Purely out of curiosity, did you consider making them a part of the Value enum more seriously? Let's say that Value didn't implement Copy and you modeled these atomics as a variant of `Arc<AtomicI64>` instead of this map, would that be equivalent? What would the implications of that design decision be?